multimap/src/main/java/org/infinispan/multimap/impl/function/set/SMIsMember.java

```diff
@@ -0,0 +1,79 @@
+package org.infinispan.multimap.impl.function.set;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import org.infinispan.commons.marshall.AdvancedExternalizer;
+import org.infinispan.functional.EntryView;
+import org.infinispan.multimap.impl.ExternalizerIds;
+import org.infinispan.multimap.impl.SetBucket;
+
+/**
+ * Serializable function used by
+ * {@link org.infinispan.multimap.impl.EmbeddedSetCache#set}
+ * to operator on Sets.
+ *
+ * @author Vittorio Rigamonti
+ * @see <a href="http://infinispan.org/documentation/">Marshalling of
+ *      Functions</a>
+ * @since 15.0
+ */
+public final class SMIsMember<K, V> implements SetBucketBaseFunction<K, V, List<Long>> {
+   public static final AdvancedExternalizer<SMIsMember> EXTERNALIZER = new Externalizer();
+   private final V[] values;
+
+   public SMIsMember(V... values) {
+      this.values = values;
+   }
+
+   @Override
+   public List<Long> apply(EntryView.ReadWriteEntryView<K, SetBucket<V>> entryView) {
+      var result = new ArrayList<Long>();
+      if (values.length == 0) {
```

Comment:
I guess we could handle this on the invocation of the function, without needing to submit a remote request?